app/views/publish_data/pages/service_manual.html.erb

```diff
@@ -33,7 +33,7 @@
             <a href="#what-is-service-performance" class="">What is Service Performance?</a>
           </li>
           <li>
-            <a href="#kpis" class="">The KPIs</a>
+            <a href="#kpis" class="">The metrics</a>
```

Comment:
@luciawhittaker could you please change `a href="#kpis"` to `a href="#metrics"` ?